code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -83,8 +83,12 @@
 		qdel(src)
 		return
 	src << "<span class='userdanger'>You're down, but not quite out. You'll be back on your feet within a minute or two.</span>"
-	spawn(rand(600,900))
+	spawn(rand(300,400))
 		if(src)
+			for(var/mob/dead/observer/ghost in player_list)
+				if(src.real_name == ghost.real_name)
```

Comment:
>see this 
>have to go check wand code to see if it does the same thing
>it does

oh geez how has this not been a problem yet? admins must be pretty good at keeping people from copycat names